test/test_runner.go

```diff
@@ -212,7 +214,18 @@ func (r *TestRunner) RunTest(script string, funcName string) (result *Result, er
 		})
 	}()
 
-	_, inter, err := r.parseCheckAndInterpret(script)
+	program, inter, err := r.parseCheckAndInterpret(script)
+	if err != nil {
+		return nil, err
+	}
+
+	err = r.runTestInit(inter)
+	if err != nil {
+		return nil, err
+	}
+
+	script = replaceImports(program.Program, script)
```

Comment:
With the `replaceImports` function, we convert imports from `StringLocation`, to their corresponding `AddressLocation`, e.g.

```cadence
import Test
import FooContract from "../contracts/FooContract.cdc"
...
```

Becomes
```cadence
import Test
import FooContract from 0x0000000000000005
...
```